samples/databases/wide-world-importers/wwi-ssdt/wwi-ssdt/WebApi/Stored Procedures/InsertBuyingGroupsFromJson.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE PROCEDURE [WebApi].[InsertBuyingGroupsFromJson](@BuyingGroups NVARCHAR(MAX), @UserID int)
+WITH EXECUTE AS OWNER
+AS BEGIN
+	INSERT INTO Sales.BuyingGroups(BuyingGroupName,LastEditedBy)
+			OUTPUT  INSERTED.BuyingGroupID
```

Comment:
Change INSERTED to inserted otherwise it will fail on a case-sensitive server.

Reply:
Updated in 03-14 commit: https://github.com/Microsoft/sql-server-samples/pull/367/commits/9c4601a8c2bd27dd252288fa4d162e78ea5e500b